sdcm/nemesis.py

```diff
@@ -461,6 +461,34 @@ def disrupt_nodetool_decommission(self, add_node=True):
             finally:
                 if new_node:
                     new_node.running_nemesis = None
+        return new_node
+
+    def disrupt_nodetool_seed_decommission(self, add_node=True):
+        def update_seed_provider(_seed_address):
+            seed_provider = [dict(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
+                                  parameters=[dict(seeds=seed_address)])]
+            for node in self.cluster.nodes:
+                node.patch_scylla_yaml(seed_provider=seed_provider)
+
+        if len(self.cluster.seed_nodes) < 2:
+            raise UnsupportedNemesis("To running seed decommission the cluster must contains at least 2 seed nodes")
+        if not self.target_node.is_seed:
+            self.target_node = random.choice(self.cluster.seed_nodes)
+        target_seed_idx = self.cluster.seed_nodes.index(self.target_node)
+        self.log.info(f"Removing the seed node in index '{target_seed_idx}' from seed nodes list")
+        self.cluster.seed_nodes.pop(target_seed_idx)
+        self.target_node.is_seed = False
+
+        seed_address = ",".join([node.ip_address for node in self.cluster.seed_nodes])
+        update_seed_provider(_seed_address=seed_address)
+
+        new_seed_node = self.disrupt_nodetool_decommission(add_node=add_node)
+        if not new_seed_node.is_seed:
+            new_seed_node.is_seed = True
+            if new_seed_node not in self.cluster.seed_nodes:
```

Comment:
it's a time to make `sdcm.cluster.BaseScyllaCluster.seed_nodes|seed_nodes_ips||non_seed_nodes` dynamic so you won't need to update the list
when @juliayakovlev  developed those methods, there was an assumption that the list of the seed nodes never changes

Reply:
Can I open a separate PR that does this or you want in the current PR?

Reply:
Done